controllers/servers/host_definer/common/settings.py

```diff
@@ -0,0 +1,37 @@
+STORAGE_API_VERSION = 'storage.k8s.io/v1'
+CSINODE_KIND = 'CSINode'
+CSI_IBM_BLOCK_API_VERSION = 'csi.ibm.com/v1'
+CSI_IBM_BLOCK_GROUP = 'csi.ibm.com'
+VERSION = 'v1'
+HOSTDEFINITION_PLURAL = 'hostdefinitions'
+HOSTDEFINITION_KIND = 'HostDefinition'
```

Comment:
```suggestion
HOST_DEFINITION_PLURAL = 'hostdefinitions'
HOST_DEFINITION_KIND = 'HostDefinition'
```
const names don't have to be that similar to their values